portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,23 +28,60 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     public ArrayList<String> list = new ArrayList<String>();
+    ArrayList<String> comments = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    list.add("Abi");
-    list.add("Patchaiyappan");
-    list.add("19");
-    String json = "{";
-    json += "\"FirstName\": ";
-    json += "\"" + list.get(0) + "\"";
-    json += ", ";
-    json += "\"LastName\": ";
-    json += "\"" + list.get(1) + "\"";
-    json += ", ";
-    json += "\"Age\": ";
-    json += "\"" + list.get(2) + "\"";
-    json += "}";
     response.setContentType("application/json");
+    String json = new Gson().toJson(comments);
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+
+    // creates arrayList of comments
+    comments.add(text);
+
+    //Datastore
+    String com = request.getParameter("text-input");
```

Comment:
```suggestion
    String commentText = request.getParameter("text-input");
```
nit: make variable names descriptive, so I can tell what value they hold